helm-sources-staging/templates/ingress.yaml

```
@@ -4,12 +4,27 @@ metadata:
   name: {{ required "Missing value DATAFABRIKKEN_INGRESSNAME" .Values.DATAFABRIKKEN_INGRESSNAME }}
   namespace: {{ required "Missing value NAMESPACE" .Values.NAMESPACE }}
   annotations:
-    nginx.ingress.kubernetes.io/rewrite-target: /
+    kubernetes.io/ingress.class: "gce"
     kubernetes.io/ingress.global-static-ip-name: {{ required "Missing value DATAFABRIKKEN_INGRESS_IP_NAME" .Values.DATAFABRIKKEN_INGRESS_IP_NAME }}
 spec:
+  ingressClassName: nginx
   rules:
   - http:
       paths:
+      - path: /veiviser-dele-data
+        pathType: ImplementationSpecific
+        backend:
+          service:
+            name: datafabrikken-veiviser-dele-data
+            port:
+              number: 8080
+      - path: /veiviser-dele-data/*
+        pathType: ImplementationSpecific
+        backend:
+          service:
+            name: datafabrikken-veiviser-dele-data
+            port:
+              number: 8080
```

Er det nødvendig med begge? Er ikke `/veiviser-dele-data/*` automatisk med i `/veiviser-dele-data` ved bruk av `pathType: Prefix`?
```suggestion
      - path: /veiviser-dele-data
        pathType: Prefix
        backend:
          service:
            name: datafabrikken-veiviser-dele-data
            port:
              number: 8080
```

---

Ja ser at gce støtter det fra 1.21.3 og oppover. Så kan endre til Prefix. Trodde det var kun nginx som støttet det.